packages/wallet-specs/src/store.ts

```diff
@@ -225,9 +228,11 @@ export class Store implements IStore {
   // PRIVATE
 
   private signState(state: State): SignedState {
+    const { privateKey } = this.getEntry(getChannelId(state.channel));
```

Comment:
I think these changes will need to be applied to the `storage/store.ts` in `xstate-wallet` as well?

Reply:
Can we simplify the codebase for the moment and only define one `store.ts`?

Reply:
The only real difference between the two `store.ts` is that the `xstate-wallet` has an implementation of `sendMessage` using `postMessage`.

If `wallet-specs` has a `store` that has a configurable messaging we could just use that for now.

Reply:
Can `xstate-wallet` extend the `Store` class from `wallet-specs`?